### ISSUE TITLE
Data types with arrays or lists are JSON-interpreted as "array of array of ..."

### ISSUE THREAD

**Issue by schteff** [stoicflame/enunciate#487]
Using Enunciate 2.6 my data types which wrap lists of other data types are interpreted as an array with small (paired) sub arrays of the other data type in the JSON examples. 

Example:
```java
@XmlRootElement
public class Counties {
	public List<County> counties;
	...
}

public class County {
	/**
	 * @documentationExample Foo
	 * @documentationExample Bar
	 */
	public String name;

	/**
	 * @documentationExample Foo2
	 * @documentationExample Bar2
	 */
	public String key;
	...
}
```

Resulting JSON example from enunciate:
```json
{
  "counties" : [ [ {
    "name" : "Foo",
    "key" : "Foo2"
  }, {
    "name" : "Bar",
    "key" : "Bar2"
  } ], [ {
    "name" : "Bar",
    "key" : "Bar2"
  }, {
    "name" : "Bar",
    "key" : "Bar2"
  } ] ]
}
```
See the extra inner arrays with pairs of Countys. [[{...},{...}][{...},{...}]]

The "data type" property in the enunciate docs for Counties is also generated as "array of array of County".

The XML examples (and the actual API calls) work fine, this only happens for the JSON examples in the enunciate docs. 

The generated swagger "Model" is:
```
Counties {
counties (Array[Inline Model 1], optional): 
}
Inline Model 1 [
json_County
]
json_County {
name (string, optional): ,
key (string, optional): 
} 
```

**Comment by stoicflame** [comment]
Thanks for the report.

**Comment by stoicflame** [comment]
Fixed with #514 and #518.

**Comment by stoicflame** [comment]
[Enunciate 2.7 has been released.](https://github.com/stoicflame/enunciate/releases/tag/v2.7.0)
